packages/mux-player-react/src/index.tsx

```diff
@@ -17,6 +17,11 @@ interface GenericEventListener<T extends Event = CustomEvent> {
   (evt: T): void;
 }
 
+export interface MuxPlayerCSSProperties extends CSSProperties {
```

Comment:
Since CSS Custom properties are inheritable via scope, we want to treat them as type-valid for any React element. Ideally, we'd enumerate all of our CSS vars in the type declaration, but it may be fine to use a similar "buckshot" templated version like you do here. For more, see: https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors

Here's an example that is produced for media-chrome's react type definitions:

```ts
import type * as CSS from 'csstype'; // A dependency from `@types/react`

// other crud...

declare module 'csstype' {
  interface Properties {
    // Should add generic support for any CSS variables
    [index: `--${string}`]: any;
  }
}
```